tpm/tss-minimal/src/handle.rs

```diff
@@ -0,0 +1,135 @@
+pub use esys_sys::ESYS_TR;
+
+use std::fmt;
+
+use crate::{private, EsysContext};
+
+pub trait TpmResource: private::Sealed {
```

Comment:
If you have references to the TCG spec sections for each of these top-level items, it would be useful to have them in doc comments, like in pkcs11-sys or mqtt-proto in the new repo.

Reply:
b6035269a38d312b5260ef910b858e0b31bfe249